src/examples/menu/Size.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Col, Row } from '@zendeskgarden/react-grid';
+import { Menu, Item } from '@zendeskgarden/react-dropdowns.next';
+
+const Example = () => (
```

Comment:
Let's repeat the side-by-side from the original in order to drive home the sizing differentiation.

Reply:
Should be updated now. :)